src/menus/EditMenus.cpp

```diff
@@ -898,9 +898,18 @@ void OnJoin(const CommandContext &context)
    auto &tracks = TrackList::Get(project);
    auto &selectedRegion = ViewInfo::Get(project).selectedRegion;
    auto &window = ProjectWindow::Get(project);
-
-   for (auto wt : tracks.Selected<WaveTrack>())
-      wt->Join(selectedRegion.t0(), selectedRegion.t1());
+   const auto progress = BasicUI::MakeProgress(
+      XO("Pre-processing"), XO("Rendering Time-Stretched Audio"), 0);
```

Comment:
I see a repetition of these strings and would like to see a convenience function to make the progress indicator, so the strings occur in just one place

Reply:
Addressed - see proposal in more recent commits.